src/sensors/distance/ultrasound/i2c/SRF08.cpp

```diff
@@ -46,16 +46,14 @@ unsigned int SRF08::getDistance()
     mRuntime.i2cEndTransmission();
 
     mRuntime.i2cRequestFrom(mAddress, kNumberOfBytesToRequest);
-    if (mRuntime.i2cAvailable() == 0)
+    if (!mRuntime.i2cAvailable())
```

Comment:
:warning: **readability-implicit-bool-conversion** :warning:
implicit conversion 'int' -> bool
```suggestion
    if (!```
```suggestion
    if (! == 0```